source/css/scss/app/components/credit-summary-lightbox.scss

```diff
@@ -10,23 +10,35 @@
 		overflow-x: scroll;
 		background-color: $white;
 
+		table tr:nth-of-type(even) {
+			background-color: $platinum;
```

Comment:
mild preference for using kiva-prefixed color variable names, e.g. $kiva-bg-darkgray, $kiva-stroke-gray, $kiva-text-dark